packages/persisted-query-lists/README.md

```diff
@@ -1,3 +1,118 @@
 # `@apollo/persisted-query-lists`
 
-Part of an upcoming Apollo GraphOS feature.
+## Setup
+
+Install the package:
+
+```sh
+npm install @apollo/persisted-query-lists
+```
+
+## Usage
+
+### `generatePersistedQueryIdsAtRuntime`
+
+Helper function passed to [Persisted Query Link](https://www.apollographql.com/docs/react/api/link/persisted-queries)
+that generates query hashes at runtime without the use of a manifest file.
```

Comment:
Possibly worth explaining how this differs from createPersistedQueryLink alone? The main point is that it does not fall back to "try to register the operation", and that it sorts the top-level definitions (hmm, the code could use a comment pointing out that it does that to match what the manifest generator code does).

Reply:
Added a sentence to explain this better. Let me know what you think.